src/data/polynomial/lifts.lean

```diff
@@ -258,7 +258,7 @@ by simp only [lifts, map_alg_eq_map, alg_hom.mem_range, ring_hom.coe_of, ring_ho
 /-- If `p` lifts and `(r : R)` then `r • p` lifts. -/
 lemma smul_mem_lifts {p : polynomial S} (r : R) (hp : p ∈ lifts (algebra_map R S)) :
   r • p ∈ lifts (algebra_map R S) :=
-by { rw mem_lifts_iff_mem_alg at hp ⊢, exact subalgebra.smul_mem (map_alg R S).range hp r }
+by rw mem_lifts_iff_mem_alg at hp ⊢; exact subalgebra.smul_mem (map_alg R S).range hp r
```

Comment:
Was this change actually necessary? I think the former style is somewhat preferred since it makes it more clear that there's only one active goal in the course of the proof.

Reply:
I thought that was the problem, but probably I misunderstood something...

Reply:
bors r-

Reply:
It seems I do not know (or I can not) how to take it off the queue to undo the change.